src/flamenco/vm/fd_vm_context.c

```diff
@@ -172,9 +172,24 @@ fd_vm_translate_vm_to_host_private( fd_vm_exec_context_t *  ctx,
   ulong end_addr = start_addr + sz;
 
   ulong host_addr = 0UL;
+
+  /* Note: vm gap shift is calculated when a memory region is created. However,
+     it is always == 63 unless the memory region is writable and gapped. This is
+     onlt the case for the stack region in memory which has a calculated value
+     of 12 for the vm_gap_shift. */
+  ulong vm_gap_shift = 0UL;
```

Comment:
Could you add permalinks for where these are in the Agave source?

Reply:
Yeah was doing that as you sent the message haha